test/fixtures_s3_test.go

```diff
@@ -27,25 +26,19 @@ const (
 type FixturesS3Suite struct {
 	suite.Suite
 	ctx        context.Context
-	logger     log.Logger
-	loader     fixtures.FixtureLoader
 	bucketName string
 }
 
 func (s *FixturesS3Suite) SetupSuite() []suite.Option {
-	s.ctx = context.Background()
-
 	return []suite.Option{
 		suite.WithLogLevel("debug"),
 		suite.WithConfigFile("test_configs/config.fixtures_s3.test.yml"),
 	}
 }
 
 func (s *FixturesS3Suite) SetupTest() (err error) {
-	s.logger = s.Env().Logger()
+	s.ctx = context.Background()
```

Comment:
Why not use the env ctx here?

Reply:
the context is not used in any "boot" functionality, so there is no need to use the environment/application context